tests/neural_compressor/test_optimization.py

```diff
@@ -194,6 +223,11 @@ def eval_func(model):
                 data_collator=default_data_collator,
             )
 
+            quantizer = IncQuantizer(
+                q8_config, eval_func=eval_func, calib_dataloader=trainer.get_eval_dataloader(), train_func=train_func
```

Comment:
Why do we also need a `calib_dataloader` for quantization aware training ? (in addition to the training function)

Reply:
In PyTorch 1.13 or above, prepare_fx function need a parameter: [exmaple_input](https://github.com/pytorch/pytorch/blob/dc00bb51b8d370bf3891f0edb2c6e0c2914e329a/torch/ao/quantization/quantize_fx.py#L271), so the dataloader will be used in static quantization and quantization-aware training.

Reply:
Was the choice of a dataloader made to have consistency with the calibration dataloader needed for PTQ ? From a user perspective I would think it would make more sense to feed one input sample

Reply:
Yes, we support the example_inputs parameter in https://github.com/huggingface/optimum-intel/pull/96.